crates/oxc_language_server/src/main.rs

```diff
@@ -346,6 +442,37 @@ impl LanguageServer for Backend {
 }
 
 impl Backend {
+    /// Request the workspace configuration from the client
+    /// and return the options for each workspace folder.
+    /// The check if the client support workspace configuration, should be done before.
+    async fn request_workspace_configuration(&self, uris: Vec<Uri>) -> Vec<Option<Options>> {
+        let length = uris.len();
+        let config_items = uris
+            .into_iter()
+            .map(|uri| ConfigurationItem {
+                scope_uri: Some(uri),
+                section: Some("oxc_language_server".into()),
+            })
+            .collect::<Vec<_>>();
+
+        let Ok(configs) = self.client.configuration(config_items).await else {
+            debug!("failed to get configuration");
+            // return none for each workspace folder
+            return vec![None; length];
+        };
```

Comment:
```suggestion
        let configs = match self.client.configuration(config_items).await {
            Ok(config) => config,
            Err(e) => {
                debug!("failed to get configuration {e:?}");
                return vec![None; length];
            }
        };
```

i think  it would be ideal that the error is logged here?